feature-detects/event/oninput.js

```diff
@@ -0,0 +1,57 @@
+/*!
+{
+  "name": "onInput Event",
+  "property": "oninput",
+  "notes": [{
+    "name": "MDN article",
+    "href": "https://developer.mozilla.org/en-US/docs/Web/API/GlobalEventHandlers.oninput"
+  },{
+    "name": "WHATWG spec",
+    "href": "http://www.whatwg.org/specs/web-apps/current-work/multipage/common-input-element-attributes.html#common-event-behaviors"
+  },{
+    "name": "Detecting onInput support",
+    "href": "http://danielfriesen.name/blog/2010/02/16/html5-browser-maze-oninput-support"
+  }],
+  "authors": ["Patrick Kettner"],
+  "tags": ["event"]
+}
+!*/
+/* DOC
+
+`oninput` tests if the browser is able to detect the input event
+
+*/
+define(['Modernizr', 'docElement', 'createElement', 'testStyles', 'hasEvent'], function( Modernizr, docElement, createElement, testStyles, hasEvent ) {
+
+  Modernizr.addTest('oninput', function() {
+    var input = createElement('input');
+    input.setAttribute("oninput", "return");
+
+    if (hasEvent('oninput', docElement) || typeof input.oninput == "function") {
```

Comment:
Running into an issue with this reporting false always... shouldn't this be `hasEvent('input', docElement)`?